enacts/flex_fcst/maproom.py

```diff
@@ -657,7 +470,7 @@ def make_map(proba, variable, percentile, threshold, start_date, lead_time):
             mapr_u.make_adm_overlay(
                 adm_name=adm["name"],
                 adm_geojson=calc.sql2GeoJSON(adm["sql"], GLOBAL_CONFIG["db"]),
-                adm_clor=adm["color"],
+                adm_color=adm["color"],
```

Comment:
typo from other splitting PRs that applied to all Maprooms but could not test on fcst until now.